WalletWasabi.Gui/Tabs/SettingsViewModel.cs

```diff
@@ -192,6 +201,14 @@ public override bool OnClose()
 			Network.RegTest
 		};
 
+		public IEnumerable<string> FeeDisplayFormats => new[]
+		{
+			"Satoshis per byte",
```

Comment:
I see that this string is used multiple times. Wouldn't it be better to create a global variable and use it rather than hard coding it each time it is needed.

Reply:
I don't think so. Would look ugly here.